pandas/tests/io/test_excel.py

```diff
@@ -103,7 +67,7 @@ def get_csv_refdf(self, basename):
         dfref = read_csv(pref, index_col=0, parse_dates=True, engine='python')
         return dfref
 
-    def get_excelfile(self, basename):
+    def get_excelfile(self, basename, ext):
```

Comment:
More verbose doing this, but with the parametrization of Test Classes performed further down in the module I think it's better to explicitly pass `ext` as an argument rather than creating subclasses that bind their own `ext` variable